fem/bilinearform.hpp

```diff
@@ -101,7 +138,9 @@ class BilinearForm : public Matrix
 public:
    /// Creates bilinear form associated with FE space @a *f.
    /** The pointer @a f is not owned by the newly constructed object. */
-   BilinearForm(FiniteElementSpace *f);
+   BilinearForm(FiniteElementSpace *f,
+                AssemblyLevel assembly_level = AssemblyLevel::FULL,
+                int elem_batch = 1);
```

Comment:
I'm not convinced by the `elem_batch` appearing in the constructor, this parameter is more for optimization purpose and is too exposed to my taste here. I think that having it initialized by default to `1` with `SetBatchSize(int elem_batch)` method would be more appropriate.